tfe/data_source_ssh_key_test.go

```diff
@@ -0,0 +1,51 @@
+package tfe
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccTFESSHKeyDataSource_basic(t *testing.T) {
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckTFESSHKeyDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccTFESSHKeyDataSourceConfig(rInt),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.tfe_ssh_key.foobar", "name", fmt.Sprintf("ssh-key-test-%d", rInt)),
+					resource.TestCheckResourceAttr(
+						"data.tfe_ssh_key.foobar", "organization", fmt.Sprintf("terraform-test-%d", rInt)),
+					resource.TestCheckResourceAttrSet("data.tfe_ssh_key.foobar", "id"),
+				),
+			},
+		},
+	})
+}
+
+func testAccTFESSHKeyDataSourceConfig(rInt int) string {
+	return fmt.Sprintf(`
+resource "tfe_organization" "foobar" {
```

Comment:
The HCL in the tests and docs can be run through `terraform format` to make sure it is all consistent.

Reply:
Yeah I know, but I'm trying to avoid that as it means copy/pasting the content every time you work on a test or the docs. Not really a nice workflow IMO, but I will update them nevertheless.

Reply:
Yeah I know, she pointed me to that as well 👍